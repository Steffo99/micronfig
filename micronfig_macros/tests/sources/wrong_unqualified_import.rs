use std::path::PathBuf;

micronfig::config! {
	GARASAUTO: String > PathBuf,
}

fn main() {
	std::env::set_var("GARASAUTO", "./auto");
	println!("{:#?}", GARASAUTO());
}
