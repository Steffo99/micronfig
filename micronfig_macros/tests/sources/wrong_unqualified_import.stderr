error[E0412]: cannot find type `PathBuf` in this scope
 --> tests/sources/wrong_unqualified_import.rs:4:22
  |
4 |     GARASAUTO: String > PathBuf,
  |                         ^^^^^^^ not found in this scope
  |
  = help: consider importing one of these items:
          crate::PathBuf
          std::path::PathBuf
